### PR TITLE
Initial frontend test env deployment pipeline

### LINKED ISSUE #97: CSG frontend test deployment pipeline
## User Story
As a CSG staff member, I want to be able to see that the code updates in the UI are reflected in the test environment shortly after development branch is updated so that code deployment is automated and confirmation that the code works is easily viewable.
### Acceptance Criteria
- successfully deploy automatically to test
- don't link PR to close the ticket!
- Sends notification to Slack on completion of deployment
- IA will do code review
### Dependencies
- Coordinate GitHub secrets with CSG staff

### PR DESCRIPTION
**Pull request by jsandoval81**
### Description List
- Created a Github workflow file for triggering frontend deployments to the CSG Test environment
- Updated to frontend app logo
    - Got the green light to use the current CompactConnect logo
    - Also will make notification links more contextual

### Draft Todo
- [x] Add repo secrets to Github for frontend Test environment _**(CSG)**_

Closes #97

### CODE REVIEW

File: .github/workflows/webroot-deploy-test.yml [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      - name: Build Vue app
+        env:
+          NODE_ENV: production
+          BASE_URL: ${{ env.BASE_URL }}
+          VUE_APP_DOMAIN: ${{ env.VUE_APP_DOMAIN }}
+          VUE_APP_ROBOTS_META: ${{ env.VUE_APP_ROBOTS_META }}
+          VUE_APP_API_STATE_ROOT: ${{ env.VUE_APP_API_STATE_ROOT }}
+          VUE_APP_API_LICENSE_ROOT: ${{ env.VUE_APP_API_LICENSE_ROOT }}
+        run: yarn build
+        working-directory: ./webroot
```
Comment (jlkravitz) [inline .github/workflows/webroot-deploy-test.yml:97]: Where are these env vars stored?

**Review by jlkravitz — APPROVED** [review]
one question on env vars but otherwise looks good and ready to merge after I meet with @isabeleliassen on Friday to populate the secrets.

File: .github/workflows/webroot-deploy-test.yml (hunk 1 above)
Comment (jsandoval81) [reply .github/workflows/webroot-deploy-test.yml:97]: Up in the `env:` section near line 32.

**Comment by jlkravitz** [comment]
@jsandoval81 added the secrets with @isabeleliassen ...let me know if it looks alright.

**Comment by jsandoval81** [comment]
@jlkravitz I don't think I can see the secret names in this repo, which is probably expected. If they're correct then things will work; and if they're close then they should be easily corrected if we see any pipeline errors.

**Comment by jlkravitz** [comment]
Got it! @isabeleliassen  good to merge this

**Pull request merged**